opencanary/logger.py

```diff
@@ -152,7 +152,8 @@ def __init__(self, config, handlers, formatters={}):
             exit(1)
 
         # Check if ignorelist is populated
-        self.ignorelist = config.getVal('ip.ignorelist', default='')
+        self.ip_ignorelist = config.getVal('ip.ignorelist', default='')
```

Comment:
can we change to `default=[]` so that it is a list as we expect

Reply:
your welcome
done